kotlinx-coroutines-core/jvm/src/scheduling/Tasks.kt

```diff
@@ -9,8 +9,19 @@ import kotlinx.coroutines.internal.*
 import java.util.concurrent.*
 
 
-// Internal debuggability name + thread name prefixes
-internal const val DEFAULT_SCHEDULER_NAME = "DefaultDispatcher"
+/**
+ * Name of the default scheduler.
+ *
+ * Customization provided for when there's no control over used dispatchers/schedulers (i.e when
+ * [Dispatchers.Default] is used within third-party libraries), or when overall default configuration
+ * is acceptable and only the default scheduler name needs to be changed.
+ *
+ * Useful for filtering log messages.
+ */
+@JvmField
```

Comment:
Added `@JvmField` for consistency with all of the other defined properties, not sure if it makes any difference in this setting. Also not sure if it can break binary compatibility somehow :thinking: